rtcp.c

```diff
@@ -942,14 +942,15 @@ uint32_t janus_rtcp_get_remb(char *packet, int len) {
 	if(packet == NULL || len == 0)
 		return 0;
 	janus_rtcp_header *rtcp = (janus_rtcp_header *)packet;
-	if(rtcp->version != 2)
-		return 0;
 	/* Get REMB bitrate, if any */
 	int total = len;
+	int offset = 0;
 	while(rtcp) {
+                if(rtcp->version != 2)
+                        break;
```

Comment:
Looks like there still is some weird indentation in the above two lines, could you fix that?

Reply:
done... I think. Now it looks good in less and the patch